zetaclient/chains/solana/signer/signer.go

```diff
@@ -152,19 +152,20 @@ func (signer *Signer) TryProcessOutbound(
 
 	case coin.CoinType_Gas:
 		if cctx.IsWithdrawAndCall() {
-			executeTxGetter, err := signer.prepareExecuteTx(ctx, cctx, height, logger)
-			if err != nil {
-				logger.Error().Err(err).Msgf("TryProcessOutbound: Fail to sign execute outbound")
-				return
-			}
 			incrementNonceTxGetter, err := signer.prepareIncrementNonceTx(ctx, cctx, height, logger)
 			if err != nil {
 				logger.Error().Err(err).Msgf("TryProcessOutbound: Fail to sign increment_nonce outbound")
 				return
 			}
-
-			txGetter = executeTxGetter
 			fallbackTxGetter = incrementNonceTxGetter
+
+			executeTxGetter, err := signer.prepareExecuteTx(ctx, cctx, height, logger)
+			if err != nil {
```

Comment:
Wondering if we should check the error, 
Fro example ,  If the `prepareExecuteTx` fails at 
```
	message, err := hex.DecodeString(cctx.RelayedMessage)
	if err != nil {
		return nil, errors.Wrapf(err, "decodeString %s error", cctx.RelayedMessage)
	}
```

Would we continue with the same logic, for `err!=nil`?